src/python/director/transformUtils.py

```diff
@@ -186,23 +172,18 @@ def frameFromPositionAndRPY(position, rpy):
     rpy specified in degrees
     '''
 
-    rpy = [math.radians(deg) for deg in rpy]
-
-    angle, axis = botpy.roll_pitch_yaw_to_angle_axis(rpy)
-
-    t = vtk.vtkTransform()
-    t.PostMultiply()
-    t.RotateWXYZ(math.degrees(angle), axis)
-    t.Translate(position)
-    return t
+    rpy = np.radians(rpy)
+    mat = transformations.euler_matrix(rpy[0], rpy[1], rpy[2])
+    mat[:3,3] = position
+    return getTransformFromNumpy(mat)
```

Comment:
Previous method: uses botpy's RPY to angle axis, then VTK's angle axis to homogeneous transform
New method: use rpy to homogeneous transform conversion from transformations.py